### PR TITLE
Add autocharge vehicle management

### PR DESCRIPTION
**Pull request by frenck**
# Proposed Changes

> (Describe the changes and rationale behind them)

Autocharge lets a car identify itself over ISO 15118 instead of needing an RFID token, and the charger keeps a separate auth list for it. Since #441 this library can be told about a vehicle through the `/vehicle/tokenfound` websocket topic, but had no way to manage the list that token is matched against. These methods mirror the existing RFID token ones: list, add, delete.

Also adds `rfid_token(uid)` for fetching a single RFID token, which the web interface has and this library did not.

The payload shapes come from the web interface's own request building code rather than guesswork: `POST` takes `EvccId`, `Alias` and `Authorize`, `DELETE` addresses the vehicle by its EVCC ID, and the list comes back wrapped in a `VehicleTokens` envelope.

Worth being explicit about test coverage here. The charger I have available runs with `Iso15118CommunicationEnable` off, so its vehicle list endpoint answers with a bare `null` rather than an empty list. That behaviour is verified against real hardware and is why `vehicle_tokens()` treats a null body as an empty list instead of failing. The add and delete paths are unit tested only, since there is no way to exercise them without autocharge enabled.

`/config/auth/vehicle-latest` is deliberately left out. It returns 403 on a charger without ISO 15118, and the web interface parses its timestamp with `new Date()`, which accepts both ISO strings and epoch milliseconds. Without a response to look at, picking one would be a guess baked into a typed model.

## Related Issues

> ([Github link][autolink-references] to related issues or pull requests)

Follows up on #441, which added the vehicle token websocket topic.

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

File: src/peblar/peblar.py [hunk 1]
```diff
@@ -95,6 +97,19 @@ class _RfidTokenListEnvelope(TypedDict):
...
+    """Single vehicle payload returned by the autocharge list endpoint."""
+
+    EvccId: str
+    Alias: str
+
+
+class _VehicleTokenListEnvelope(TypedDict):
+    """Autocharge vehicle list response envelope."""
+
+    VehicleTokens: list[_VehicleTokenPayload]
```
[the remark this answers is not in the document]
Comment (frenck) [reply src/peblar/peblar.py]: Correct, the type was claiming more than the charger guarantees. Fixed in the latest commit:

```python
VehicleTokens: NotRequired[list[_VehicleTokenPayload] | None]
```

`NotRequired` for the absent case, `| None` for the null case, which between them cover all three shapes the tests exercise (`null` body, `{}`, and `{"VehicleTokens": null}`). The runtime behaviour was already right, this just stops the annotation contradicting it.

Left `_RfidTokenListEnvelope` alone deliberately. It is a different endpoint and has never been observed answering with anything but a populated `Tokens` list, so loosening it would be speculation rather than a fix.
